### ISSUE TITLE
Use progress bars to indicate docker operation progress

### ISSUE THREAD

**Issue by arschles** [deis/deisrel#104]
After #89 is done, `deisrel` will do multiple docker operations at once. Instead of just printing the raw event stream out to stdout, this issue is for using a progress bar like [github.com/cheggaaa/pb](godoc.org/github.com/cheggaaa/pb).

Note that the number of steps each operation will require is not known a priori, so we may have to use a different type of progress bar - one that indicates the name of each operation, but not a definitive endpoint, for example.

**Comment by Joshua-Anderson** [comment]
deisrel no longer uses docker, so this is no longer needed.
